openshift_images/using_images/images-other-jenkins-agent.adoc

```diff
@@ -26,6 +26,22 @@ The Maven and Node.js Jenkins agent images provide Dockerfiles for the Universal
 Use and extend an appropriate agent image version for the your of {product-title}. If the `oc` client version that is embedded in the agent image is not compatible with the {product-title} version, unexpected behavior can result.
 ====
 
+{product-title} also provides sample PodTemplates defined in the {product-title}
+Jenkins image that illustrates how to use these agent images with the
+Jenkins Kubernetes plug-in.
+
+The four PodTemplates defined are:
```

Comment:
Please replace all instances of "PodTemplates" with "`PodTemplates` <object type>" OR "pod templates".